lib/shopify_api/app.ex

```diff
@@ -23,23 +23,27 @@ defmodule ShopifyAPI.App do
           scope: String.t()
         }
 
+  @per_user_query_params ["grant_options[]": "per-user"]
+
   require Logger
+
+  alias ShopifyAPI.AssociatedUser
   alias ShopifyAPI.AuthRequest
+  alias ShopifyAPI.AuthToken
   alias ShopifyAPI.JSONSerializer
+  alias ShopifyAPI.UserToken
 
   @doc """
     Generates the install URL for an App and a Shop.
   """
-  @spec install_url(__MODULE__.t(), String.t()) :: String.t()
-  def install_url(%__MODULE__{} = app, domain) when is_binary(domain) do
-    query_params = [
-      client_id: app.client_id,
-      scope: app.scope,
-      redirect_uri: app.auth_redirect_uri,
-      state: app.nonce
-    ]
+  @spec install_url(__MODULE__.t(), String.t(), boolean()) :: String.t()
+  def install_url(app, domain, per_user \\ false)
```

Comment:
🍎 I wonder if the `per_user` variable name should mirror the option instead `use_online_tokens`

Reply:
Yeah, makes sense